apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js

```diff
@@ -55,19 +54,18 @@ const IdeaForm = ({
 						style={{
 							maxWidth: '36rem',
 							margin: '1.5rem auto 1.5rem',
```

Comment:
could we wrap this inside of an `atoms.Box` and pass in these as `props` to that component instead of using inline styles?